src/worker/core/player/shouldRetire.ts

```diff
@@ -9,13 +9,19 @@ import type {
 const shouldRetire = (
 	p: Player<MinimalPlayerRatings> | PlayerWithoutKey<MinimalPlayerRatings>,
 ): boolean => {
-	const age = g.get("season") - p.born.year;
-
+	const age = g.get("season") -p.born.year;
+	
```

Comment:
Don't worry about it for this PR, but it's weird to see formatting like this because Prettier should be running every commit...